server/apiserver/argoserver.go

```diff
@@ -259,7 +268,8 @@ func (as *argoServer) Run(ctx context.Context, port int, browserOpenFunc func(st
 	httpL := tcpm.Match(cmux.HTTP1Fast())
 	grpcL := tcpm.Match(cmux.Any())
 
-	go eventServer.Run(as.stopCh)
+	go as.runConfigMapWatcher(as.stopCh)
```

Comment:
this will cause lots of logs on k8s 1.27+ like https://github.com/argoproj/argo-workflows/issues/11657

Reply:
Are there any solutions already? Should we change it to informer? I am referring to controller

Reply:
Supposedly changing to an Informer will fix it. It's on my to-do list as a contributor abandoned a PR I was reviewing. Should hopefully get to it this week (per my status, the, uh, kidney stone has made things a little harder)

Reply:
> this will cause lots of logs on k8s 1.27+ like #11657

also, to be clear, it's the `RetryWatcher` on line 459 below that will cause this more specifically

Reply:
> Supposedly changing to an Informer will fix it. It's on my to-do list as a contributor abandoned a PR I was reviewing. Should hopefully get to it this week (per my status, the, uh, kidney stone has made things a little harder, pun unintended)

Okay, I'm looking forward to your PR. Will your PR solve the problem of argo server re-downloading configmap?

Reply:
No, just #11657 for the Controller